crates/bevy_render/src/camera/projection.rs

```diff
@@ -103,12 +103,12 @@ impl CameraProjection for OrthographicProjection {
     fn update(&mut self, width: f32, height: f32) {
         match (&self.scaling_mode, &self.window_origin) {
             (ScalingMode::WindowSize, WindowOrigin::Center) => {
-                let half_width = width / 2.0;
-                let half_height = height / 2.0;
-                self.left = -half_width;
+                let half_width = (width / 2.0).round();
+                let half_height = (height / 2.0).round();
+                self.left = half_width - width;
                 self.right = half_width;
                 self.top = half_height;
-                self.bottom = -half_height;
+                self.bottom = half_height - height;
```

Comment:
I don't really see what this change does? Why is this different from `-half_height`?

Reply:
For odd `height`, `half_height` will be rounded up, resulting `top-bottom = 2*half_height = height+1` with the original assignment of `bottom`. The changed calculation ensures that `top-bottom` is still equal to `height` (and the same for left/right/width).

Reply:
Ah, got it. Can you please add this as a comment to the code, so then it doesn't get reverted later?

Reply:
Done